diagnostics/eulerian_storm_track/notes

```diff
@@ -0,0 +1 @@
+time for daily is "days since start year"
```

Comment:
What is this purpose of this notes file?

Reply:
It was an internal file, I removed it.